scripts/install.sh

```diff
@@ -308,6 +338,11 @@ fi
 
 echo -e "\n\nFHIR Works is deploying. A fresh install will take ~20 mins\n\n"
 ## Deploy to stated region
+lambdaLatencyThreshold=$lambdaLatencyThreshold \
```

Comment:
is '\' at the end necessary ? We don't need to export these variables ?

Reply:
I had to use two \\\\ to show a \\.
I meant instead of `lambdaLatencyThreshold=$lambdaLatencyThreshold \\` can we just do  `lambdaLatencyThreshold=$lambdaLatencyThreshold` ?

Reply:
Don't mean to speak for Niroj here but I can speak to the backslashes.  All the backslashes here make all these variables part of the `yarn run....` command on line 346.  They are only visible as envvars to the yarn process.

Reply:
That makes sense now. That's why export can be omitted. Thanks Jamal

Reply:
I was using \ to make sure that all the commands are run as a single command. Those parameters are env for serverless deploy command.